content/docs/getting_started/uninstall.md

```diff
@@ -87,7 +87,7 @@ Use the `osm` CLI to uninstall the OSM control plane from a Kubernetes cluster.
 1. OSM controller resources (deployment, service, config map, and RBAC)
 1. Prometheus, Grafana, Jaeger, and Fluentbit resources installed by OSM
 1. Mutating webhook and validating webhook
-1. CRDs installed/required by OSM: [CRDs for OSM](https://github.com/openservicemesh/osm/tree/main/charts/osm/crds). This will also remove all Custom Resources related to those CRDs. Refer to [Removal of OSM Cluster Wide resources](#remove-osm-cluster-wide-resources) for more details
+1. The conversion webhook fields patched by OSM to the CRDs installed/required by OSM: [CRDs for OSM](https://github.com/openservicemesh/osm/tree/main/cmd/osm-bootstrap/crds) will be unpatched. Refer to [Removal of OSM Cluster Wide resources](#remove-osm-cluster-wide-resources) for more details
```

Comment:
I think we should refer to the main branch-specific references in documentation meant for the main branch. When we cut the release, we will only update the references in the release branch. I know this process is not being followed consistently, but we should do so. It would be consistent with how our image tags in the OSM repo reflect the branch the code is targeted to.

Reply:
I agree, that makes sense. Then we should revisit which version https://docs.openservicemesh.io/ renders by default since it uses the main branch of this repo currently by default where I know we've said in the past it should be the latest release.

Reply:
Yes, docs.openservicemesh.io should point to the latest release as suggested in https://github.com/openservicemesh/osm-docs/issues/191. We should fix this for v1.0 as tagged.